geopyspark/tests/base_test_class.py

```diff
@@ -4,4 +4,4 @@
 
 
 class BaseTestClass(unittest.TestCase):
-    geopysc = GeoPyContext(master="local[*]", appName="test")
+    geopysc = GeoPyContext(master="local[1]", appName="test")
```

Comment:
This may be a point of discussion.  The tests appeared to be failing on Travis due to resource constraints.  I did some tests earlier, and running this with `local[32]` would spawn as many as 18 unique (gateway) JVMs.